exporter/fileexporter/config.go

```diff
@@ -44,6 +44,9 @@ type Config struct {
 	// Compression Codec used to export telemetry data
 	// Supported compression algorithms:`zstd`
 	Compression string `mapstructure:"compression"`
+
+	// FlushInterval number of seconds between flushes.
+	FlushInterval int `mapstructure:"flush_interval"`
```

Comment:
Typically we will use a `time.Duration` for measures of time. This allows users to pick more aggressive intervals if they want, or easily use very long ones. e.g. `100ns`, `2h`

Reply:
I  have changed it to Duration.